src/PriceIndicator/PriceIndicator.styles.ts

```diff
@@ -1,6 +1,10 @@
 import styled from 'styled-components';
 import { motion } from 'framer-motion';
 
+export const StylePricingContainer = styled.div`
+  transform: translateY(${p => p.theme.spacing.small});
```

Comment:
This will offset the margins set below, so when we use it, it will honour the spacings given there